src/features/components/AddBook.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const AddBook = () => (
+  <>
+    <h2 className="add-book-form">ADD NEW BOOK</h2>
+    <div className="add-book-form">
+      <input className="book-input" placeholder="Book title" />
+      <select className="category-select" name="categories" id="categories">
+        <option defaultValue="">Category</option>
+        <option value="action">Action</option>
+        <option value="science-fiction">Science Fiction</option>
+        <option value="economy">Economy</option>
+      </select>
+      <input className="add-book-button" type="submit" value="ADD BOOK" />
+    </div>
+  </>
+);
+
+export default AddBook;
```

Comment:
- [ ] The structure of the files should follow the ducks pattern.

---------------------------------------------Yours------------------------------------------

![react7](https://user-images.githubusercontent.com/55635977/130774645-5517e61e-ad71-4c8d-bda8-b1a3206a66b2.png)

---------------------------------------------Original/Requirement------------------------------------------

![react8](https://user-images.githubusercontent.com/55635977/130774730-c5edfad6-c8ae-4996-bca0-30d25b3b97f3.png)

Reply:
File structure updated as requested

Reply:
Well solved, however you are missing configure.js file